### PR TITLE
workaround vendor filepath hack

### PR DESCRIPTION
**Pull request by shayne-fletcher**
- MR https://gitlab.haskell.org/ghc/ghc/-/merge_requests/8516:
  - Temporary hack to vendor in filepath breaks cabal parsing in `ghc-lib-gen`
  - Work around

### CODE REVIEW

**Comment by shayne-fletcher** [comment]
cc @garyverhaegen-da @akrmn @sofiafaro-da @cocreature

File: ghc-lib-gen/src/Ghclibgen.hs
Comment (shayne-fletcher) [inline ghc-lib-gen/src/Ghclibgen.hs:358]: surprisingly some redundant whitespace has crept in which is getting removed here incidentally

**Review by akrmn — APPROVED** [review]
Work around a workaround :) 
Thanks!

**Comment by shayne-fletcher** [comment]
> Work around a workaround :) Thanks!

@akrmn  😆 

btw, i forget what it's called and i don't have privilege to check but there's a label you can apply so that, now you've approved, if/when the checks pass it will auto-land

**Comment by moisesackerman-da** [comment]
@shayne-fletcher hm I couldn't find the label, but I do have the auto-merge button that does the same, I'll remember to use that next time

**Pull request merged**
